### ISSUE TITLE
Model server reports illegal reflective access error on start up of coffee editor

### ISSUE THREAD

**Issue by martin-fleck-at** [eclipse-emfcloud/emfcloud-modelserver#8]
Directly after launching the coffee editor, the following editor is reported on the console.

```
root INFO Theia app listening on http://localhost:3000.
root ERROR ModelServerBackendContribution: WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$2 (file:/home/lucas/Git/coffee-editor/web/coffee-server/build/workflow-modelserver-example-1.2.0-SNAPSHOT-glsp.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by cbos** [comment]
This is related to google/guice#1133

**Comment by martin-fleck-at** [comment]
Google Guice solved that issue with version 5.0! However, it does not happen in our jar anymore, probably due to some dependency update where the reflective access was happening. So I am closing this issue for now.
